Solution1/ClassLibrary1/NumericUpDownLight.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace ClassLibrary1
+{
+    public class NumericUpDownLight : Control
+    {
+        private int _value = 0;
+        private EventHandler _valueChanged = null;
+        private HorizontalAlignment _textAlign = HorizontalAlignment.Left;
+        private Color _mouseHoverBackColor = Color.Blue;
+
+        private readonly CustomTextBox TextBox = new CustomTextBox();
```

Comment:
We have code [style rules](https://grapecity.atlassian.net/wiki/spaces/C1/pages/2933166541/WF+Coding+style+guidelines)
document.
private filed should use "_" prefix.